cmd/push.go

```diff
@@ -374,7 +380,7 @@ func (p *PushCmdParams) getSelectedAccounts() ([]string, error) {
 	return []string{}, nil
 }
 
-func multiRequest(nc *nats.Conn, report *store.Report, operation string, subject string, reqData []byte, respHandler func(srv string, data interface{})) int {
+func multiRequest(nc *nats.Conn, report *store.Report, operation string, subject string, reqData []byte, respHandler func(srv string, data interface{}), timeout int) int {
```

Comment:
I would put `timeout` before `respHandler` to keep handler as the last param.

Reply:
since that is effectively something related to the connection/etc (we need scatter gather in the go client), I put it after the connection, also refactored some of the param names so that we can understand what is going on in there.